x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/formula.tsx

```diff
@@ -100,28 +100,11 @@ export const formulaOperation: OperationDefinition<
     return [
       {
         type: 'function',
-        function: 'mapColumn',
+        function: 'mathColumn',
         arguments: {
           id: [columnId],
           name: [label || defaultLabel],
-          exp: [
-            {
-              type: 'expression',
-              chain: currentColumn.references.length
-                ? [
-                    {
-                      type: 'function',
-                      function: 'math',
-                      arguments: {
-                        expression: [
-                          currentColumn.references.length ? `"${currentColumn.references[0]}"` : ``,
-                        ],
-                      },
-                    },
-                  ]
-                : [],
-            },
-          ],
+          expression: [currentColumn.references.length ? `"${currentColumn.references[0]}"` : ``],
```

Comment:
Seems like this is causing the failing test with an empty formula which is annoying - I tried to fix it using `staticColumn`, but it's missing separate id/name params. We could add those, not sure whether there's a more elegant solution.

Reply:
The best solution I've found is to use `mapColumn` with an empty expression.